SystemInformer/tokprp.c

```diff
@@ -527,14 +527,21 @@ PPH_STRING PhGetElevationTypeString(
 
     PhInitializeStringBuilder(&sb, 13);
 
-    PhAppendStringBuilder2(&sb, IsElevated ? L"Yes" : L"No");
-
-    if (ElevationType == TokenElevationTypeFull)
-        PhAppendStringBuilder2(&sb, L" (Full)");
-    else if (ElevationType == TokenElevationTypeLimited)
-        PhAppendStringBuilder2(&sb, L" (Limited)");
+    if (ElevationType)
+    {
+        PhAppendStringBuilder2(&sb, IsElevated ? L"Yes" : L"No");
+
+        if (ElevationType == TokenElevationTypeFull)
+            PhAppendStringBuilder2(&sb, L" (Full)");
+        else if (ElevationType == TokenElevationTypeLimited)
+            PhAppendStringBuilder2(&sb, L" (Limited)");
+        else if (ElevationType == TokenElevationTypeDefault)
+            PhAppendStringBuilder2(&sb, L" (Default)");
+    }
     else
-        PhAppendStringBuilder2(&sb, L" (Default)");
+    {
+        PhAppendStringBuilder2(&sb, L"Unknown");
```

Comment:
Generally elsewhere in the UI when something is "N/A" or "Unknown" we leave it blank. But, I don't have a strong feeling one way or another.

Reply:
In my opinion, it should be used one value (Unknown / N/A) over the whole GUI; which one -> for me it doesnt matter.
I have choosed Unknown, because its used in `IDD_OBJTOKEN` of SystemInformer.rc as default.